packages/governance/src/contractGovernor.js

```diff
@@ -61,7 +61,6 @@ const validateQuestionFromCounter = async (zoe, electorate, voteCounter) => {
 /**
  * @typedef {StandardTerms} ContractGovernorTerms
  * @property {TimerService} timer
- * @property {Instance} electorateInstance
```

Comment:
Please remove "electorate" from line 72 below, and add the following paragraph after line 84:
```
 * The electorate that will vote on governance questions is itself a governed
 * parameter. Its value is available from the publicFacet using
 * getElectorateInstance(). When creating new questions, we use
 * getUpdatedPoserFacet() to inform the electorate about the new question.
```

Reply:
Thanks for the close read!

I'm force-pushing with that comment change.